study.md

```diff
@@ -43,16 +44,17 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+For an Array there is a predetermined size of the data structure, which can be returned instantaneously.
+
+For an enumerable there is no way to know what sort of structure is being operated on. Therefore in order to get the size of the enumerable it must go through and count as it goes. Does that mean 'count' on an infinite enumerable will count up forever?
```

Comment:
I'd imagine that trying to invoke `count` on an infinite enumerator will result in an infinite loop.